src/views/catalog/EnableInstanceTypeTechPreviewModal/EnableInstanceTypeTechPreviewModal.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import { Trans } from 'react-i18next';
+import instanceTypeTechPreviewModalImage from 'images/instance-type-tech-preview-modal-img.svg';
+
+import { useIsAdmin } from '@kubevirt-utils/hooks/useIsAdmin';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { usePreviewFeatures } from '@kubevirt-utils/hooks/usePreviewFeatures/usePreviewFeatures';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import {
+  Button,
+  ButtonVariant,
+  Modal,
+  ModalVariant,
+  Split,
+  SplitItem,
+  Stack,
+  StackItem,
+  Text,
+  TextVariants,
+  Title,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+type EnableInstanceTypeTechPreviewModalProps = {
+  navigateToCatalog: () => void;
+};
+
+const EnableInstanceTypeTechPreviewModal: FC<EnableInstanceTypeTechPreviewModalProps> = ({
+  navigateToCatalog,
+}) => {
+  const { t } = useKubevirtTranslation();
+
+  const { instanceTypesEnabled, loading, toggleInstanceTypesFeature, error } = usePreviewFeatures();
```

Comment:
on each modal open it will make the 3 network calls?

Reply:
no, only if CM is not found on the first-time admin visits this tab it will run 4 network calls, 1 for watch CM and 3 for create, and if exists only 1 call for watching the CM